README.md

```diff
@@ -147,3 +149,55 @@ field(:create_booking, :string) do
   resolve(&TestResolver.run/3)
 end
 ```
+
+## Error Formatting
+
+The package includes GraphQL specification compliant error formatting. When enabled, validation errors from constraints or transform failures are formatted consistently.
```

Comment:
This package is internal, isn't it?  Can we link to RFC?

Reply:
package is public :)